src/CompressionPolicy.cs

```diff
@@ -16,24 +16,20 @@
  */
 
 using System.IO;
+using System.IO.Compression;
 
 namespace Apache.NMS.ActiveMQ
 {
-    /// <summary>
-    /// Default Compression policy for NMS.ActiveMQ uses the built in GZipStream
-    /// to compress and decompress the message body.  This is not compatible with
-    /// compression used by ActiveMQ so users should use this with caution.
-    /// </summary>
     public class CompressionPolicy : ICompressionPolicy
     {
         public Stream CreateCompressionStream(Stream data)
         {
-			return new Ionic.Zlib.ZlibStream(data, Ionic.Zlib.CompressionMode.Compress);
+            return new GZipStream(data, CompressionMode.Compress);
```

Comment:
Based on documentation from microsoft and some information on SO, this should be sufficient. The comment above has me concerned. Why not just use GZipStream from the start? Was this some special functionality added for a different compression mode?